src/Assets/Asset.php

```diff
@@ -837,6 +847,44 @@ public function is_vendor(): bool {
 		return $this->is_vendor;
 	}
 
+	/**
+	 * Gets the asset bases for the request, both directory and URL.
+	 *
+	 * @since 1.2.3
+	 *
+	 * @return array
+	 */
+	protected function get_bases(): array {
+		$key = md5( serialize( [ WP_CONTENT_DIR, WP_CONTENT_URL ] ) );
```

Comment:
Just trying to be extra careful this time. Can we base the `$key` on things that should be able to change during runtime ? Constants should not change during runtime.

I would `$key = md5( serialize( [ plugins_url(), get_stylesheet_directory_uri()] ) );`

To make our life easier while testing, we could include everything in the key if we want.